components/OperatePage/FAQ.jsx

```diff
@@ -0,0 +1,165 @@
+      },
+      {
+        title: 'How do I install Pearl?',
+        desc: (
+          <>
+            Visit the
+            {' '}
+            <Link href="/operate#download" className="text-purple-600">
+              Pearl download section
+            </Link>
+            {', '}
+            download the app, and follow the installation guide provided.
+          </>
+        ),
+      },
+      {
+        title: 'What are the system requirements for Pearl?',
+        desc: (
+          <>
+            Minimal system requirements for Mac devices:
+            <ul className="list-disc ml-6">
+              <li>
+                <Link
+                  href="https://docs.docker.com/docker-for-mac/install/"
+                  className="text-purple-600"
+                  target="_blank"
+                >
+                  Docker Desktop↗
+                </Link>
+                {' '}
+                running. During Alpha and Beta phases only.
+              </li>
+              <li>800 MB of free RAM (suggested 1 GB)</li>
+              <li>1.3 GB disk space</li>
+            </ul>
+          </>
+        ),
+      },
+      {
+        title: 'What happens if I forget my password?',
+        desc: (
+          <>
+            Pearl uses Safe smart wallets. Safe smart wallets are designed so
+            that, if you forget your password, you can restore access to your
+            funds using a backup wallet. This backup wallet acts as a second
+            signer on your Safe. To learn more about this third party product
+            and try to regain access, go to&nbsp;
+            <Link
+              href="https://safe.global/"
+              className="text-purple-600"
+              target="_blank"
+            >
+              safe.global
+            </Link>
+            , connect your backup wallet, and then access the Safe.
+          </>
+        ),
+      },
+    ],
+  },
+  {
+    name: 'General',
+    list: [
+      {
+        title: 'Can I run an agent without using the app?',
+        desc: (
+          <>
+            Yes, for more technical control and flexibility, you can set up and
+            run agents manually. Visit
+            {' '}
+            <Link
+              href="https://operate.olas.network/"
+              className="text-purple-600"
+            >
+              operate.olas.network
+            </Link>
+            {' '}
+            to get started.
+          </>
+        ),
+      },
+      {
+        title: 'How do I earn OLAS tokens?',
+        desc: 'Earn OLAS by staking OLAS and running agents on your computer.',
+      },
+      {
+        title: 'Where can I get support if I encounter issues?',
+        desc: (
+          <>
+            There is no official support, but the
+            {' '}
+            <Link
+              target="_blank"
+              href="https://discord.com/channels/899649805582737479/1169275451089367131"
```

Comment:
should we point to the new channel? https://valory-workspace.slack.com/archives/C06HNEWREG3/p1716803582527929?thread_ts=1716803167.118759&cid=C06HNEWREG3